assets/js/recommender.js

```diff
@@ -0,0 +1,35 @@
+'use strict'
+
+let NUM_RECOMMENDED_PAGES = 5
```

Comment:
Should we let this value be optionally configurable in `_config.yml`, with a default value of 5 instead?

Reply:
Maybe for a start, we'll control the number of links centrally. And if there is need for variety in the future, we can add the value as a config.